app/Notifications/ProjectExportNotification.php

```diff
@@ -60,6 +68,39 @@ public function toMail($notifiable)
                     ->line('Thank you for using our application!');
     }
 
+    /**
+     * Get the array representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function toDatabase($notifiable)
+    {
+        return [
+            'message' => "Project[$this->projectName] has been exported successfully.",
+        ];
+    }
+
+    /**
+     * Get the broadcastable representation of the notification.
+     * @param  mixed  $notifiable
+     * @return BroadcastMessage
+     */
+    public function toBroadcast($notifiable)
+    {
+        $timestamp = Carbon::parse(now()->addSecond()->toDateTimeString());
+        return new BroadcastMessage(array(
+            'notifiable_id' => $notifiable->id,
+            'notifiable_type' => get_class($notifiable),
+            'data' => $this->toDatabase($notifiable),
+            'notifiable' => $notifiable,
+            'read_at' => null,
+            'created_at' => $timestamp->diffForHumans(),
+            'updated_at' => $timestamp->diffForHumans(),
+            'file_path'  => url(Storage::url('exports/'.basename($this->path))),
```

Comment:
Please add space after concatenate operator like ( url(Storage::url('exports/' . basename($this->path))) )